docs/docs/components/reanimated_swipeable.md

```diff
@@ -81,27 +81,29 @@ an argument.
 
 ### `renderLeftActions`
 
-method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
-This map describes the values to use as inputRange for extra interpolation:
+accepts a function which receives the following arguments:
 
-SharedValue: [startValue, endValue]
+- `progress` value representing swiping progress relative to the width of the returned element.  
```

Comment:
```suggestion
- `progress` - a shared value representing the swiping progress relative to the width of the returned element.
```

a tiny grammar change

Reply:
>shared value

Wouldn't `SharedValue` work better?

> the

Not sure if it's neccessary, the element it's refferring to is already unique ("the returned element"), and i don't think `progress` is a unique thing itself, as one swipeable has left action `progress`, and right action `progress`. 

> `  `

The double spaces are intentional, they ensure a line break.
While i think `\` would be clearer, it unfortunately doesn't work with our current setup.

Reply:
> Wouldn't `SharedValue` work better?

I think that both are fine, `SharedValue` is more straightforward.

> Not sure if it's necessary, the element its referring to is already unique (...)

It is not referring to the element itself. Also I believe we can keep it.

> The double spaces are intentional, they ensure a line break.

What about making it a sub-bullet instead?

Reply:
> I think that both are fine, SharedValue is more straightforward.

Alright
>It is not referring to the element itself. Also I believe we can keep it.


Alright



>What about making it a sub-bullet instead?

Is this what you meant?

![image](https://github.com/user-attachments/assets/88ff41a3-db44-4920-b8fc-3a2f56f16063)

Reply:
I thought about 2 sub-bullets (2 last sentences)

Reply:
![image](https://github.com/user-attachments/assets/3d14fb6f-9e8b-497a-ad3e-242a914a3323)

Reply:
That's better. We just have to make adjustments from other comments 😅

Reply:
>That's better. We just have to make adjustments from other comments 😅

I think the current version ([9845471](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/9845471eab4539a1a6d9f4164ab3483a159b296e)) has all the adjustments that were requested.